spec/features/toc_links_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Table of Contents links", type: :feature, js: true do
```

Comment:
I would make this feature test less specific. `features/toc_links_spec` -> `features/toc_spec`.